### PR TITLE
SpectralAnomalyScore for preinstalled function catalog

### PR DESCRIPTION
**Pull request by sedgewickmm18**

### CODE REVIEW

File: iotfunctions/anomaly.py [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+logger = logging.getLogger(__name__)
+PACKAGE_URL = 'git+https://github.com/ibm-watson-iot/functions.git@'
+_IS_PREINSTALLED = True
+
+
+class SpectralAnomalyScore(BaseTransformer):
+    '''
+    Employs spectral analysis to extract features from the time series data and to compute zscore from it
+    '''
+    def __init__(self, input_item, windowsize, output_item):
```
Comment (ankit-jha) [inline iotfunctions/anomaly.py]: If the library allows do declare defaults we can do that

`def __init__(self, input_item, windowsize=12, output_item):`

File: iotfunctions/anomaly.py [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+# which accompanies this distribution, and is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# *****************************************************************************
+
+'''
+The Built In Functions module contains preinstalled functions
+'''
+
+import datetime as dt
```
Comment (ankit-jha) [inline iotfunctions/anomaly.py]: #import datetime as dt
#import time
#from collections import OrderedDict
#import skimage as ski
#import re
#import warnings
#import json

Good to remove unused imports

File: iotfunctions/anomaly.py (hunk 1 above)
Comment (sedgewickmm18) [reply iotfunctions/anomaly.py]: good idea - btw, I've modified the code to fix the kmeans approach.
you might notice that there is some code overlap, maybe we can simplify things..

**Comment by sedgewickmm18** [comment]
merging so that Ankit can continue

**Pull request merged**